packages/@webex/plugin-meetings/src/breakouts/events.ts

```diff
@@ -1,29 +1,45 @@
 // eslint-disable-next-line import/prefer-default-export
-import {ClientEvent, NewMetrics} from '@webex/internal-plugin-metrics';
+import {ClientEvent} from '@webex/internal-plugin-metrics';
 
 const breakoutEvent: {
-  onBreakoutMoveRequest: (eventInfo: any) => void;
-  onBreakoutMoveResponse: (eventInfo: any) => void;
-  onBreakoutJoinResponse: (eventInfo: any) => void;
-  postMoveCallAnalyzer: (event: ClientEvent['name'], eventInfo: any) => void;
+  onBreakoutMoveRequest: (eventInfo: any, submitClientEvent: any) => void;
```

Comment:
Well because i had to remove the singleton approach, I can't just import that function in this file and call it at the end.
Now I have to access the submit method via this.webex.internal.newMetrics, and since this.webex is not available in this file, AND because of the way the metrics logic for breakouts was before, this was the quickest solution...

I don't like it but at the same time I don't want to spend time refactoring breakouts related code if I can avoid it.